.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "addon-sdk"]
-	path = addon-sdk
-	url = https://github.com/mozilla/addon-sdk.git
```

Comment:
I see. The interesting bit is that JPM doesn't need anything to produce the XPI from the add-on SDK package anymore?